src/hb/paint_extents.rs

```diff
@@ -23,6 +23,20 @@ impl hb_extents_t {
     pub fn is_empty(&self) -> bool {
         self.x_min > self.x_max
     }
+
+    pub fn union_(&mut self, o: &hb_extents_t) {
```

Comment:
Maybe `unite` or even `r#union`?

Reply:
Ah, I realized the reason this is in harfbuzz is because `union` is a keyword... But I guess that's not the case in Rust, so we don't need to use it there.